cmd/capdctl/main.go

```diff
@@ -25,10 +25,14 @@ import (
 
 	"github.com/pkg/errors"
 	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/rest"
+	"k8s.io/kubernetes/pkg/kubectl/scheme"
```

Comment:
k8s.io/kubernetes has been explicitly described as "not designed for import" -- do you think there's another way to do this?